build.gradle.kts

```diff
@@ -110,33 +48,42 @@ val packageVersion: String by project
 group = pluginGroup
 version = packageVersion
 
-logger.lifecycle("Building Amazon Ion $pluginVersion for ${descriptor.platformType} ${descriptor.platformVersion}")
+logger.lifecycle("Building Amazon Ion $pluginVersion")
 
 dependencies {
     intellijPlatform {
-        create(descriptor.platformType.acronym, descriptor.platformVersion) // https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html#setting-up-intellij-platform
-        bundledPlugins(descriptor.bundledDependencies)
+        // This should be equivalent to the lowest version that we build against.
+        intellijIdeaCommunity(supportedSinceIdeVersion.let(::branchToProductReleaseVersion))
+
+        bundledPlugins(listOf(
+            "com.intellij.java",
+            "org.jetbrains.kotlin",
+        ))
         pluginVerifier()
         instrumentationTools()
 
         testFramework(TestFrameworkType.Platform)
     }
 
     testImplementation("org.junit.jupiter:junit-jupiter:5.7.1")
-    testCompileOnly("junit:junit:4.13")
+    testCompileOnly("junit:junit:4.13.1")
```

Comment:
This looks like it needs some explanation, why are we so specific about the version number?

Reply:
`4.13.0` is affected by a CVE. I'll add a comment to that effect.